src/index.js

```diff
@@ -337,6 +493,7 @@ class ClientAuthOIDC {
 
   /**
    * Redirects the current window to the given uri.
+   *
```

Comment:
Yeah, it looks strange, doesn't it :( It has to do with some horrible browser quirk - on some browsers, the `window.location.href` redirect only works if you `return false` from that function. (See https://stackoverflow.com/a/6109823/525338 for an example)